src/hooks/Guilds/contracts/useDecodedCall.ts

```diff
@@ -161,7 +161,24 @@ export const decodeCall = async (
       };
   }
   if (!matchedContract) {
-    return null;
+    decodedCall = {
+      callType: SupportedAction.RAW_TRANSACTION,
+      from: call.from,
+      to: call.to,
+      value: call.value,
+      function: null,
+      args: {},
+      optionalProps: {
+        data: call.data,
+      },
+    };
+
+    return {
+      id: `action-${Math.random()}`,
+      decodedCall,
+      contract: null,
+      approval: null,

```

Comment:
Should we include `approval: call.approval || null` just in case? or at this point is impossible that an approval call reaches this if sentence?